CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ was committed to the `master` branch. This is not necessarily the date that
 the code was deployed.
 
 ## [Unreleased]
+### Added
+- Added landing page so page will load when there are no installations selected or present (CU-gurbtf).
```

Comment:
Can you please specify that this is the Chatbot landing page? Just to distinguish it from the Heartbeat pages. Thanks!

Reply:
Will do!